consensus/merge/merge.go

```diff
@@ -208,7 +207,7 @@ func (s *Merge) FinalizeAndAssemble(config *chain.Config, header *types.Header,
 	if config.IsPrague(header.Time) {
 		header.RequestsHash = outRequests.Hash()
 	}
```

Comment:
I think we can add isthmus check here too

Reply:
Hmm. I guess the above `s.Finalize` will already check for it and return empty array for the `outRequests` so we don't need to check here